### PR TITLE
Fix admin dashboard new ui

### PR DESCRIPTION
**Pull request by sapk**
capture

### CODE REVIEW

File: public/less/_admin.less [hunk 1]
```diff
@@ -25,4 +25,29 @@
...
+    	padding: 20px;
+    }
+    .admin-table{
+        .table{
+            width: 100%;
+        }
+    }
+    dl.admin-dl-horizontal {
+    	padding: 20px;
+		margin: 0;
```
Comment (Unknwon) [inline public/less/_admin.less]: Please fix indent.

File: templates/admin/dashboard.tmpl [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
+        </div>
+        <br>
+        
+        
+        <h4 class="ui top attached header">
+          {{.i18n.Tr "admin.dashboard.operations"}}
+        </h4>
+        <div class="ui attached table segment">
+   
                                 <div class="admin-table">
```
Comment (Unknwon) [inline templates/admin/dashboard.tmpl]: Indent here as well.

**Comment by Unknwon** [comment]
LGTM.

**Comment by pecigonzalo** [comment]
:+1:

**Pull request merged**